Kmeans_python/fit.py

```diff
@@ -30,5 +106,64 @@ def fit(X_train, k):
     >>> centers, labels = fit(X, 2)
     
     """
+    if not ((isinstance(X_train, pd.DataFrame)) | (isinstance(X_train, np.ndarray))):
+        raise ValueError("Invalid input type for samples."
+                         " X_train must be pandas dataframe or a numpy array.")
+        
+    if n_init <= 0:
+        raise ValueError("Invalid number of initializations."
+                         " n_init=%d must be bigger than zero." % n_init)
+
+    if max_iter <= 0:
+        raise ValueError("Number of interations should be a positive integer."
+                         " max_iter=%d must be bigger than zero." % max_iter)
+    
+    if k < 1:
+        raise ValueError("Invalid number of clusters."
+                         " k=%d must be an integer bigger than or equal to 1." % k)
```

Comment:
```suggestion
                         " k=%d must be an integer greater than or equal to 1." % k)
```

Reply:
Subjective grammatical connotation. I followed the one in sklearn kmeans source code : https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/cluster/_kmeans.py